internal/provider/models.go

```diff
@@ -26,6 +26,8 @@ const (
 	CockroachAPIKey string = "COCKROACH_API_KEY"
 	APIServerURLKey string = "COCKROACH_SERVER"
 	UserAgent       string = "terraform-provider-cockroach"
+	CCVanityName    string = "CC_VANITY_NAME"
+	CCUsername      string = "CC_USERNAME"
```

Comment:
It would be nice to use the same env var prefix for all the cockroach env vars the user might need to set. A bit awkward if you want the cloud on there as well though. Maybe @marksoper has a thought here?

Reply:
From the customer perspective I think it will be more clear to have all env variables related to anything Cockroach under the same namespace via the prefix "COCKROACH_". We already have a precedent for CC-only concepts like COCKROACH_API_KEY being prefixed this way, so it's tough now to try to scope env variables to CC. Plus "CC" may be ambiguous in the context of a bunch env vars from different products. @pritesh-lahoti if you feel it's important to distinguish Cloud concepts moving forward, we could consider somethign like "COCKROACH_CLOUD_", but my preference is simply "COCKROACH_" since we already have established a different pattern with COCKROACH_API_KEY.

Reply:
Done